src/drivers/spektrum_rc/spektrum_rc.cpp

```diff
@@ -50,8 +50,11 @@
 #include <uORB/topics/input_rc.h>
 
 // Snapdraogon: use J12 (next to J13, power module side)
```

Comment:
I think that better way is to be able to pass serial port from the command line. Right now it is not using it. Let me check if I can fix it.